src/fparser/two/utils.py

```diff
@@ -694,6 +694,10 @@ def match(separator, subcls, string):
                 "SequenceBase class match method argument string expected to "
                 "be a string but found '{0}'.".format(type(string)))
 
+        if separator == ' ':
+            raise InternalError(
+                "SequenceBase class match method argument separator can not "
```

Comment:
"cannot"

Reply:
Please update the docstring :raises: too.

Reply:
Modified. I also changed another 'can not' to 'cannot' and removed a spurious multiple white space removal (as this is no longer required).